src/components/AllProducts/products.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import heartIcon from "../../assets/img/logo/heartIcon.svg";
+import styles from "./styles.module.css";
+import EMPTY_IMAGE_URL from "../../assets/img/landing/Img_home_02.png";
+
+function ProductList({ products }) {
+  return (
+    <ul className={styles["all-list"]}>
+      {/* 상품 목록이 있을 경우, 상품 정보를 렌더링 */}
+      {products.length > 0 ? (
```

Comment:
없을 경우가 짧은 경우엔 없는 경우부터 처리해주시는 걸 추천드려요!
```suggestion
      {products.length === 0 ? (
```